velox/docs/functions/spark/math.rst

```diff
@@ -48,6 +48,39 @@ Mathematical Functions
 
     Returns the cosecant of ``x``.
 
+.. spark:function:: decimal_round(decimal[, scale]) -> [decimal]
```

Comment:
Should this go into https://facebookincubator.github.io/velox/functions/spark/decimal.html ?

Reply:
The round function for floating type is in https://facebookincubator.github.io/velox/functions/spark/math.html, so it was added here. But since it is a decimal special form, it looks more proper to go into https://facebookincubator.github.io/velox/functions/spark/decimal.html. Updated.